content/docs/04.5-devx/05-app-profile/02-container-deployment.md

```diff
@@ -19,21 +19,34 @@ To add a container tier to Palette Dev Engine App Profile,
 
 <br />
 
-1. Log in to the Palette Dev Engine console
+# Prerequisite
 
+* [Spectro Cloud Palette account](https://www.spectrocloud.com/get-started/)
 
-2. Select the `App Profiles` from the left ribbon menu, click `+ New App Profile` and provide the following information to the app creation wizard.
+
+## How to Deploy Container to App Profile
+
+1. Login to the Palette Dev Engine console.
```

Comment:
```suggestion
1. Login to [Palette](/devx#quickstartwithpaletteappmode).
```
This changes the language to be the same as the other How-to's related to app profile.